llarp/service/endpoint_util.cpp

```diff
@@ -37,18 +37,22 @@ namespace llarp
     void
     EndpointUtil::ExpirePendingTx(llarp_time_t now, PendingLookups& lookups)
     {
+      std::vector<std::unique_ptr<IServiceLookup>> timedout;
```

Comment:
nitpick, but probably call this `timed_out` so it doesn't look like the noun `timeout`